cb_maintenance/models/maintenance_request.py

```diff
@@ -21,18 +20,49 @@ class MaintenanceRequest(models.Model):
         default="unspecified",
         required=True,
     )
+    follower_id = fields.Many2one("res.users", readonly=True)
     category_id = fields.Many2one(related=False)
-    solved = fields.Boolean(related="stage_id.done")
     close_datetime = fields.Datetime("Close Date", readonly=True)
     maintenance_team_id_member_ids = fields.Many2many(
         "res.users",
         relation="selectable_maintenance_members",
         compute="_compute_maintenance_team_id_member_ids",
     )
     technician_user_id = fields.Many2one(
-        domain="[('id', 'in', maintenance_team_id_member_ids)]", default=False
+        compute="_compute_technician_user_id", store=True
     )
-    manager_id = fields.Many2one("res.users", string="Manager")
+    technician_id = fields.Many2one(
+        "res.partner",
+        # TODO: Add domain
+    )
+    manager_id = fields.Many2one(
+        "res.users",
+        string="Manager",
+        domain="[('id', 'in', maintenance_team_id_member_ids)]",
+        default=False,
+    )
+
+    color = fields.Integer(compute="_compute_color", store=True)
+
+    @api.depends("maintenance_type")
+    def _compute_color(self):
+        for record in self:
+            # Hardcoded colors, green for preventive and red for corrective
+            c_preventive = 10
```

Comment:
Maybe on a parameter? Maybe there is other types. What should we do with this? The color will be visible on trees and forms?

Reply:
I think its not that bad to hardcode this, we can make a decision on it and if in the future we want to change it, we can just do a PR, its not like we want this customizable. Also it is harder to read from the css or other possible places where we may use the values.
At most I would create a global variable at the top of the file so we can import it from other places.